packages/document-drive/src/server/base-server.ts

```diff
@@ -795,9 +713,54 @@ export class BaseDocumentDriveServer
     return this.documentStorage.getChildren(driveId);
   }
 
+  protected async addChild(
+    parentId: string,
+    documentId: string,
+  ): Promise<void> {
+    // TODO: check if document exists? Should that be a concern here?
```

Comment:
This should be handled by storage, as it needs to test for a variety of other issues (like circular refs).